examples/batchReceive.ts

```diff
@@ -8,15 +8,12 @@ const path = process.env[entityPath] || "";
 
 async function main(): Promise<void> {
   const client = EventHubClient.createFromConnectionString(str, path);
-  const receiver = await client.createReceiver("0", { enableReceiverRuntimeMetric: true });
-  console.log("Created Receiver for partition 0 and CG $default.");
-  let result: EventData[] = await receiver.receive(10);
+  const result: EventData[] = await client.receiveBatch("0", 10, 20, { enableReceiverRuntimeMetric: true });
```

Comment:
nit: if you already have an options object i'd probably shove the timeout in there as well and provide a default